### ISSUE TITLE
Include a standard PSF as a Filter parameter

### ISSUE THREAD

**Issue by aboucaud** [aboucaud/galcheat#109]
As proposed by @herjy, in future releases of galcheat we could try to add a standard PSF to the parameters.

The format is still to be discussed : small image, parametric representation, optical wavefront.

The important point is that is should be consistent across surveys.

**Comment by herjy** [comment]
An easy way to do this to begin with is probably to use analytic profiles with `galsim` as @thuiop suggested I think.
For now it's besides the point, but I don't think all PSFs have to use the same model or type of models, as long as the API remains the same no matter what survey we use. e.g. I would see no problem with calling something like 
```p_hsc_r = HSCSurvey.r.psf(ra, dec, t)                  # For HSC r-band PSF ```
```p_hst_f814w = HSTSurvey.f814w.psf(ra, dec, t)  # For HST f814w PSF ```
One calls a parametric profile, but the other uses an image and a distortion under the hood, that would be completely fine with me. I put `ra`, `dec` and `t`, but I don't anticipate that we'd provide full time and space coverage for varying PSFs, though if the information is available and can be used in galcheat, why not. And if not, we should just let the user know that the psf used is static.

**Comment by herjy** [comment]
I'll try to make a proposition there.

**Comment by ismael-mendoza** [comment]
Would the BTK default PSF be approrpiate for this ? ( at least for the constant version)

**Comment by herjy** [comment]
Right, actually BTK already has everything we need for that. This is making me reconsider. Should we leave it at that or provide what is basically the `get_psf` method from BTK here? 
I think that's the kind of thing that has its place in galcheat but we should think of what's the easiest.

**Comment by ismael-mendoza** [comment]
Maybe we can discuss this in the BTK meeting this Wednesday, I hesitate to include simulations from galsim in galcheat since it feels out of scope for a "cheatsheet".

**Comment by ismael-mendoza** [comment]
Axel suggests: provide psf parameters / format understood by galsim so that galsim is not a dependency. See how they do it in https://github.com/LSSTDESC/psf-weather-station
